typescript/sdk/src/deploy/utils.ts

```diff
@@ -18,3 +32,14 @@ export function getMultiProviderFromConfigAndSigner<Chain extends ChainName>(
   }));
   return new MultiProvider(chainProviders);
 }
+
+export function getChainToOwnerMap<Chain extends ChainName>(
```

Comment:
This is similar to the above issue that this function assumes that the owner is the same address on every chain

Reply:
If they're not, then they don't have to use this util but we ourselves use it enough to make it worth including I think

Reply:
I think the point was more to indicate its usefulness only in test